scripts/airgapped/openebs-images-1.9.0.txt

```diff
@@ -0,0 +1,15 @@
+quay.io/openebs/m-apiserver:1.9.0
```

Comment:
I notice that tags are repeated everywhere. In txt as well as shell scripts.
This can probably be improved eventually.

Reply:
If .txt file has the image name & image tag, then scripts need not repeat this.